package.json

```diff
@@ -41,7 +41,9 @@
     "url"
   ],
   "dependencies": {
+    "@lokesh.dhakar/quantize": "~1.4.0",
     "debug-logfmt": "~1.2.3",
+    "ndarray": "~1.0.19",
     "node-addon-api": "~8.3.0",
     "node-gyp": "~11.0.0",
     "node-vibrant": "~3.2.1-alpha.1",
```

Comment:
Do you still need this one?

Reply:
yes, check https://github.com/microlinkhq/splashy/blob/c740654d367f587b34d37a316ca213b1346ba224/src/index.js#L7